<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="enterPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lensTypeLabel">
        <property name="text">
         <string>Lens Type</string>
        </property>
        <property name="buddy">
         <cstring>lensTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="frontRadiusLabel">
        <property name="text">
         <string>Front Radius</string>
        </property>
        <property name="buddy">
         <cstring>frontRadiusLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="backRadiusLabel">
        <property name="text">
         <string>Back Radius</string>
        </property>
        <property name="buddy">
         <cstring>backRadiusLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="focalLengthLabel">
        <property name="text">
         <string>Focal Length</string>
        </property>
        <property name="buddy">
         <cstring>focalLengthLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="baseThicknessLabel">
        <property name="text">
         <string>Base Thickness</string>
        </property>
        <property name="buddy">
         <cstring>baseThicknessLineEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="lensTypeComboBox">
        <item>
         <property name="text">
          <string>Custom Lens</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spherical Lens</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="frontRadiusLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="backRadiusLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="focalLengthLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="baseThicknessLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>261</width>
      <height>74</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="distributionLabel">
        <property name="text">
         <string>Distribution</string>
        </property>
        <property name="buddy">
         <cstring>distributionComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstParameterLabel">
        <property name="text">
         <string>Parameter 1</string>
        </property>
        <property name="buddy">
         <cstring>firstParameterLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="secondParameterLabel">
        <property name="text">
         <string>Parameter 2</string>
        </property>
        <property name="buddy">
         <cstring>secondParameterLineEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="distributionComboBox">
        <item>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maxwell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstParameterLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="secondParameterLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
